### ISSUE TITLE
Feature Request: Function to change Global Datatype into a Page Datafolder

### ISSUE THREAD

**Issue by DBailey635** [Orckestra/CMS-Foundation#327]
Prompted by #310, please could we have a contextual menu item in the Data perspective that would transform a Global Datatype into a Page Datafolder? The function would need to prompt the user to select a page to attach any existing data to, before making the change.

I might be a good idea to have another function to convert from a Page Datafolder into a Global Datatype as well, but I see this as being less important.

**Comment by burningice2866** [comment]
@DBailey635 One issue with converting Global Datatype to a Page Datafolder, is what page existing data should be tied to. Should there be a dialog-prompt when converting and existing data exists, where one can select what page the existing data should relate to?

**Comment by DBailey635** [comment]
@burningice2866 Yes, it definitely needs a dialogue box to prompt for a page to attach existing data to (GD > PD). Likewise, upon reversal (PD > GD), a dialogue to warn that existing data will be detached from pages.
